SSA/Projects/InstCombine/Base.lean

```diff
@@ -216,7 +247,7 @@ namespace Op
 @[match_pattern] abbrev urem   : Nat → Op := MOp.urem   ∘ .concrete
 @[match_pattern] abbrev srem   : Nat → Op := MOp.srem   ∘ .concrete
 @[match_pattern] abbrev select : Nat → Op := MOp.select ∘ .concrete
-@[match_pattern] abbrev add    : Nat → Op := MOp.add    ∘ .concrete
+@[match_pattern] abbrev add (w : Nat) (nsw : Bool := false) (nuw : Bool := false) : Op:=  MOp.add (.concrete w) nsw nuw
```

Comment:
line is too long, also please be consistent with whitespace
```suggestion
@[match_pattern] abbrev add (w : Nat) (nsw : Bool := false) (nuw : Bool := false) : Op := 
  MOp.add (.concrete w) nsw nuw
```

The other lines are packed close together, since they are all relatively simple, uniform, and closely related. 
This definition breaks that mold. Once a definition spans more than one line, we almost always (or at least, this is my opninion, tastes on code style do differ) want to have empty lines separating it from other defs. In this case, it's probably best to yoink `add` after `const` (notice how `icmp` and `const` are also separated, because they are also different)